dev/com.ibm.ws.jaxrs.2.x.monitor/src/com/ibm/ws/jaxrs/monitor/JaxRsMonitorFilter.java

```diff
@@ -74,6 +77,115 @@ public class JaxRsMonitorFilter implements ContainerRequestFilter, ContainerResp
     private static final RestMonitorKeyCache monitorKeyCache = new RestMonitorKeyCache();
     private static final String STATS_CONTEXT = "REST_Stats_Context";
 
+    private static final ConcurrentHashMap<RestRouteKey, String> routes = new ConcurrentHashMap<>();
```

Comment:
With telemetry they use a class called RestRouteCache to encapsulate the logic for the cache.  Likely that can just be copied to this project in this package.  Hopefully it will be the exact same code.  That way we can easily compare the files to see if something changes.